index.html

```diff
@@ -738,18 +740,39 @@ <h3>Introduction</h3>
 					SVG images are not supported in the [=spine=] but can be embedded in XHTML documents.</p>
 			</section>
 
-			<section id="html-req">
-				<h3>Requirements</h3>
+			<section id="xhtml">
+				<h3>XHTML</h3>
 
-				<p>An [=eBraille content document=]:</p>
+				<p>An [=eBraille content document=] MUST be an <a data-cite="epub-33#sec-xhtml">XHTML content
+						document</a> as defined [[epub-33]] with additional constraints as defined in the following
+					subsections.</p>
 
-				<ul>
-					<li>MUST be an <a data-cite="epub-33#sec-xhtml">XHTML content document</a> [[epub-33]].</li>
-					<li>MUST NOT contain features specified in <a href="#html-no-support"></a>.</li>
-				</ul>
+				<section id="xhtml-char-encoding">
+					<h4>Character encoding</h4>
 
-				<section id="html-no-support">
-					<h3>Unsupported features</h3>
+					<p>[=eBraille content documents=] are required to provide their text content in 6- or 8-dot braille
+						characters. eBraille is not meant as a delivery format for ASCII braille or similar braille
+						encodings that require a translation to 6- or 8-dot braille characters.</p>
+
+					<p>For this reason, the text content of [=eBraille content documents=] SHOULD consist only of
+						characters in the <a href="https://www.unicode.org/charts/nameslist/c_2800.html">Braille
+							Patterns</a> [[unicode]] as well as [=ASCII whitespace=] [[infra]].</p>
+
+					<div class="note">
+						<p>This restriction only applies to the renderable text of an XHTML document as well as any
+							attributes that could be rendered depending on the display capabilities of the user's device
+							(e.g., the [^img/alt^] attribute [[html]] for images that cannot be displayed).</p>
```

Comment:
Another important one that will be used in eBraille is `abbr` on `th` elements. And perhaps we need to remind that in the [page list](https://raw.githack.com/daisy/ebraille/spec/html-char/index.html#example-page-list-with-front-and-body-matter) we explicitly require that the `title` attribute of entries is NOT in braille.

HTML defines [translatable attributes](https://html.spec.whatwg.org/multipage/dom.html#translatable-attributes), but I guess we can't refer to that?

Reply:
> HTML defines [translatable attributes](https://html.spec.whatwg.org/multipage/dom.html#translatable-attributes), but I guess we can't refer to that?

No, that's not quite the same thing. It would allow braille in lang and style attributes, for example. They don't define attributes in the same kinds of classes as elements. Maybe it would make sense to provide a more complete list but without making the list exclusive.

Reply:
Note that it does make sense in style attributes, although we probably shouldn't go there. To illustrate: in DAISY Pipeline, when it transcribes an HTML document to an HTML document with braille as text content, it goes into the style attributes and translates the "content" properties.

Reply:
Would title on abbr be another case, or how does braille software handle expansions? I always think of an AT possibly announcing the expansion, so needing tts-ready text, but does braille software provide the expansion in braille?

Reply:
Good question. I would seem logical but I really don't know. I don't know if AT even uses title on abbr for TTS.

I can also imagine it would in some cases create funny situations when an expansion is provided but the expansion is actually abbreviated because abbreviations/contractions are baked into the braille code. Perhaps it would make more sense to provide expansions in uncontracted braille.

Reply:
They can use the title but typically it's a configuration option the user has to enable, like in jaws. I'd assume a braille device would provide the same option to use the expansions in place of the default text. It's probably worth mentioning it, as I don't expect we'd want plain text used.

But I also get confused about where voicing comes into play as I'm too used to making mainstream publications more accessible. I put aria-label on the nav elements without braille text in the navigation document examples thinking an AT would use those for announcing landmarks. Is that a realistic use case for braille? In other words, we wouldn't want to impose this restriction on aria-label even though it's an attribute whose text could sometimes be rendered.